Automattic/simplenote-macos
# Interlinks Autocomplete UI: Mark III

### IMPORTANT
- [x] Ship SimplenoteInterlinks Package 1.1.0 before merging!!

### Details:
In this PR we're enhancing the Interlink Autocomplete Window so that **return** or **double click** inserts the selected note's Interlink.

@aerych Sir!! May I bug you yeeet again?
Thanks in advance!!!

Ref. #663
Ref. [Simplenote Interlink PR 2](https://github.com/Automattic/SimplenoteInterlinks-Swift/pull/2)

### Test
1. Open any note and enable Markdown support
2. Type `[keyword` (with `keyword` being a word present in any other note)
3. Select any note that shows up in your search reuslts

- [x] Verify that double clicking over any row inserts a **Simplenote Interlink™**
- [x] Verify that pressing return over any row **also** inserts a **Simplenote Interlink™**
- [x] Verify the newly inserted link is linkified (underlined, should be clickable!)
- [x] Verify that clicking over such link ends up opening the target note

### Release
`RELEASE-NOTES.txt` was updated in acd1da5 with:

> Typing `[` will now allow you to pick Interlinking Notes right from the editor
